frontend/packages/knative-plugin/integration-tests/features/eventing/create-event-sink.feature

```diff
@@ -0,0 +1,68 @@
+              And user has created or selected namespace "aut-event-sink"
+              And user has created channel "kn-channel"
+              And user is at Add page
+
+
+        @regression @odc-6390
+        Scenario: Create event sink from Add page: KES-01-TC01
+             When user clicks on Event Sink card
+              And user clicks on "Kafka Sink" card
+              And user clicks on Create Event Sink
+              And user selects Output Target as "kn-channel"
+              And user enters name as "event-sink-test1"
+              And user clicks on Create button
+             Then user will see "event-sink-test1" created in topology
+
+
+        @regression @odc-6390
+        Scenario: Create event sink from catalog page: KES-01-TC02
+            Given user is at catalog page
+             When user selects Types as Event Sinks
+              And user clicks on "Log Sink" card
+              And user clicks on Create Event Sink
+              And user selects Output Target as "kn-channel"
+              And user enters name as "event-sink-test2"
+              And user clicks on Create button
+             Then user will see "event-sink-test2" created in topology
+
+
+        @regression @odc-6390
+        Scenario: Create event sink with YAML view: KES-01-TC03
+             When user clicks on Event Sink card
+              And user clicks on "Log Sink" card
+              And user clicks on Create Event Sink
+              And user switches to YAML view
+              And user clicks on Create button
+             Then user will see "kamelet-log-sink" created in topology
+
+
+        @regression @odc-6390
+        Scenario: Create event sink from context menu: KES-01-TC04
+            Given user is at Topology page
+             When user right clicks in empty space of topology
+              And user selects "Event Sink" from Add to project
+              And user clicks on "Log Sink" card
+              And user clicks on Create Event Sink
+              And user selects Output Target as "kn-channel"
+              And user enters name as "event-sink-test3"
+              And user clicks on Create button
+             Then user will see "event-sink-test3" created in topology
+
+
+        @regression @odc-6390
+        Scenario: Connect event sink to Broker: KES-01-TC05
+            Given user created Broker "test-broker"
+              And user created "Log Sink" event sink "event-sink-test" with target as "kn-channel"
+              And user is at Topology page
+             When user right clicks on the Broker "test-broker" to open the context menu
+              And user clicks on the Add Trigger
+              And user clicks on the Subscriber dropdown on the modal
+              And user selects the Subscriber as "event-sink-test"
+              And user clicks on Add button
+             Then user will see connection between Broker "test-broker" and Event Sink "event-sink-test"
```

Comment:
This test case is wrong. We don't have event sinks available in the Add Trigger modal.
Once user right clicks on Broker or drag connector and drops it on graph, User will see `Add Event Sink` option in the context menu and after that creation flow is as usual.

Reply:
Updated